addons/website_event/static/src/js/website_event_options.js

```diff
@@ -0,0 +1,50 @@
+odoo.define('website.event_options', function (require) {
+'use strict';
+
+const { _t, qweb } = require('web.core');
+const Dialog = require('web.Dialog');
+const editor = require('web_editor.snippet.editor');
+const { EditMenuDialog } = require('website.contentMenu');
+
+editor.Class.include({
```

Comment:
This should not be done via an editor include but with a regular option ... like the option when clicking on the website menu actually. This task is probably just about extending that one actually so my guess is that the whole diff should half of this.